compiler-plugin-tests/src/test/java/io/ballerina/stdlib/graphql/compiler/ServiceValidationTest.java

```diff
@@ -1085,6 +1085,43 @@ public void testUnsupportedPrimitiveTypeAlias() {
         assertErrorMessage(diagnostic, message, 19, 6);
     }
 
+    @Test
+    public void testUnsupportedUsagesOfIdAnnotation() {
+        String packagePath = "62_invalid_usages_of_id_annotation";
```

Comment:
Should we think about changing the test function name or packagePath value to be the same? Typically, we use similar names between the test name and the resource files.